### PR TITLE
Ubiquity Journals.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Ubiquity Journals.js [hunk 1]
```diff
@@ -31,35 +31,35 @@
 	You should have received a copy of the GNU Affero General Public License
 	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
 */
-function detectWeb(doc, url) {
+function detectWeb(doc) {
```
Comment (zuphilip) [inline Ubiquity Journals.js]: `_url`

Comment (retorquere) [reply Ubiquity Journals.js]: done

**Comment by zuphilip** [comment]
Thank you!

**Comment by retorquere** [comment]
Very welcome, glad to do it.

Given how much back and forth we've had for most translators (most of which is on me, I readily acknowledge), is it still the general rule that we don't lint-fix translators unless there's also a functional change lined up? I'm thinking the lint fixes may be intimidating to people submitting PRs, and also that it won't be easy to keep it in the preferred two-commit format for such PRs (one lint, one functional) given the back-and-forths we've had.

**Comment by zuphilip** [comment]
For me, the most important PR are the ones improving or fixing existing translators, then new translators can be interesting, and just after that anything like linting PR or updating test cases. Therefore, I introduced a new label: [improvements](https://github.com/zotero/translators/labels/Improvements). It would also be nice if we could adjust the coding style, but this is for me of smaller priority.

I am quite sure that we soon have a common understanding, how to fix linting issues and therefore there will be less back-and-forth. For any new contributor I would also offer to do linting for them or just ignore them and accept the PR with Travis complaining about it.

**Comment by retorquere** [comment]
Coding style is only part of the linting though. On several occasions, the linter found actual errors (like the recent `dcterms` variable not existing on Unqualified RDF something something).

**Pull request merged**